.github/workflows/aiohttp.yml

```
@@ -0,0 +1,64 @@
+name: Aiohttp
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  test:
+    permissions:
+      contents: read # to fetch code (actions/checkout)
+
+    name: Aiohttp regression tests
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout aiohttp
+      uses: actions/checkout@v4
+      with:
+        repository: aio-libs/aiohttp
+    - name: Checkout llhttp
+      uses: actions/checkout@v4
+      with:
+        path: vendor/llhttp
+    - name: Restore node_modules cache
+      uses: actions/cache@v3
+      with:
+        path: vendor/llhttp/.npm
+        key: ubuntu-latest-node-${{ hashFiles('vendor/llhttp/**/package-lock.json') }}
+        restore-keys: ubuntu-latest-node-
+    - name: Install llhttp dependencies
+      run: npm install --ignore-scripts
+      working-directory: vendor/llhttp
+    - name: Build llhttp
+      run: make
```

Looks like we run
```suggestion
      run: make generate
```
_(https://github.com/aio-libs/aiohttp/blob/b51d5d7/Makefile#L67)_

---

I'm going to keep it to what the llhttp docs say, so it's more maintainable for them if they need to update the llhttp build steps.

---

I don't think it's a good idea — if they fix the workflow, they'll deviate from aiohttp further and we'll never get notified about that. A better workflow is to be notified if they start seeing issues, so that we'd fix that ourselves and know that our repo also needs chaning.

By the way, it occurred to me that we could have a reusable workflow hosted in the aiohttp repo and just called here.